ReSharper.FSharp/test/data/parsing/Expr - CompExpr - LetUseAndBang 02.fs

```diff
@@ -0,0 +1,8 @@
+async {
+     let! x = ()
+     and! y = ()
+     use! z = ()
+     let v = ()
+     use c = ()
```

Comment:
Please keep the tests as simple as possible. It seems that having multiple non-recursive bindings afterwards doesn't test anything that isn't covered by other tests.

Reply:
Sure. I'd like to leave use! though - I haven't seen it tested anywhere. I will look one more time

Reply:
Could it be a separate and simpler test then, please? 🙂

Reply:
I replaced this one with simple let-and inside CE.